app/models/version.rb

```diff
@@ -204,6 +204,7 @@ def update_attributes_from_gem_specification!(spec)
       metadata: spec.metadata || {},
       requirements: spec.requirements,
       built_at: spec.date,
+      rubygems_version: (spec.required_rubygems_version || '').to_s,
```

Comment:
I used `|| ''` cause thats what existed [on bundler-api repo](https://github.com/bundler/bundler-api/blob/7bf5af30c5079e4a99ab0e2646c023d41380f7ad/lib/bundler_api/update/gem_db_helper.rb#L87). I suppose it is for gemspec which have not specified `required_rubygems_version`? Checkout this commit: https://github.com/bundler/bundler-api/commit/ee311886c95da0d01cb4f8a6c2a85b1440c39402